compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -788,7 +788,12 @@ extern "C" LLVMMetadataRef LLVMRustDIBuilderCreateFile(
     const char *Directory, size_t DirectoryLen, LLVMRustChecksumKind CSKind,
     const char *Checksum, size_t ChecksumLen) {
   Optional<DIFile::ChecksumKind> llvmCSKind = fromRust(CSKind);
```

Comment:
Can you update this one too? AFAICS that's the only `Optional` we have left. Then the `#include` can be version-conditional too, and we won't be back here again if/when that's removed entirely.

Reply:
Thank you! Done.